src/py/mat3ra/notebooks_utils/pyodide/packages/torch.py

```
@@ -192,6 +186,552 @@ def patch_torch_testing():
+
+
+def patch_fairchem_deps():
+    """
+    Stub heavy dependencies that fairchem-core imports but doesn't need for inference.
+
+    This stubs: numba, ray (+ serve), wandb, torchtnt, hydra, omegaconf,
+    submitit, clusterscope, tqdm, huggingface_hub, websockets.
+    """
+    # --- numba ---
+    numba_mod = _make_stub_module("numba", submodules=["core", "core.types", "typed"])
+    numba_mod.njit = lambda *a, **k: (lambda f: f) if not a or callable(a[0]) else lambda f: f
```

Ruff (0.15.13)

[warning] Useless `if`-`else` condition

(RUF034)